controllers/reviewController.js

```diff
@@ -0,0 +1,10 @@
+const {fetchReviews} = require('../models/reviewModels')
+
+exports.getReviews = (req, res) => {
+    fetchReviews().then((reviewArr) => {
+        res.status(200).send(reviewArr)
```

Comment:
In this case I would suggest changing the name to reviews rather than reviewArr as it is more explanatory and clear.